Tests/CatalogItems/Rest/Write-RsRestCatalogItem.Tests.ps1

```diff
@@ -66,12 +67,18 @@ Describe "Write-RsRestCatalogItem" {
             VerifyCatalogItemExists -itemName 'SimpleMobileReport' -itemType 'MobileReport' -folderPath $rsFolderPath -reportServerUri $reportServerUri
         }
 
-        It "Should upload a local PBIX file" {
+        It "Should upload a local small PBIX file" {
             $itemPath = $localPath + '\SimplePowerBIReport.pbix'
             Write-RsRestCatalogItem -ReportPortalUri $reportPortalUri -Path $itemPath -RsFolder $rsFolderPath -Verbose
             VerifyCatalogItemExists -itemName 'SimplePowerBIReport' -itemType 'PowerBIReport' -folderPath $rsFolderPath -reportServerUri $reportServerUri
         }
 
+        It "Should upload a local large PBIX file" {
+            $itemPath = $localPathToLargeReport + '\LargePowerBIReport.pbix'
```

Comment:
not a good practice to have big files in the repo, please see my other comment about using a variable to define the size to use one endpoint or the other and with that you can use the existing test reports but passing a different value to the function